epoxy-paging3/src/main/java/com/airbnb/epoxy/paging3/PagingDataEpoxyController.kt

```diff
@@ -0,0 +1,102 @@
+package com.airbnb.epoxy.paging3
+
+import android.annotation.SuppressLint
+import android.os.Handler
+import androidx.paging.PagingData
+import androidx.recyclerview.widget.DiffUtil
+import com.airbnb.epoxy.EpoxyAsyncUtil
+import com.airbnb.epoxy.EpoxyController
+import com.airbnb.epoxy.EpoxyModel
+import com.airbnb.epoxy.EpoxyViewHolder
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+
+@ObsoleteCoroutinesApi
+abstract class PagingDataEpoxyController<T : Any>(
+    /**
+     * The handler to use for building models. By default this uses the main thread, but you can use
+     * [EpoxyAsyncUtil.getAsyncBackgroundHandler] to do model building in the background.
+     *
+     * The notify thread of your PagedList (from setNotifyExecutor in the pagingData Builder) must be
+     * the same as this thread. Otherwise Epoxy will crash.
+     */
+    modelBuildingHandler: Handler = EpoxyAsyncUtil.getAsyncBackgroundHandler(),
```

Comment:
The previous version was using `EpoxyController.defaultModelBuildingHandler`, why change it? same with the diffingHandler